src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/Windows/WindowsContainerSnapshotProvider.cs

```diff
@@ -195,7 +195,7 @@ private double MemoryPercentage(Func<ulong> getMemoryUsage)
         {
             if (now >= _refreshAfterMemory)
             {
-                _memoryPercentage = Math.Min(Hundred, memoryUsage / _memoryLimit * Hundred); // Don't change calculation order, otherwise we loose some precision
+                _memoryPercentage = Math.Min(Hundred, memoryUsage / _memoryLimit);
```

Comment:
Considering that the first parameter `Hundred` (defined above as `double 100.0d`), was used to 'cap' utilization at 100.0 in the previous version, wouldn't it make sense to 'cap' it at 1.0 going forward?